cmd/podman/main_remote.go

```diff
@@ -32,8 +32,12 @@ func init() {
 
 func setSyslog() error {
 	// Log to file if not using syslog
-	homeDir := homedir.Get()
-	path := filepath.Join(homeDir, ".config", "containers")
+	path := os.Getenv("XDG_CONFIG_HOME")
```

Comment:
This block for retrieving home path needs to be a helper, we're calling it a lot

Reply:
I think I've smushed it down as much as I can with the latest.  After reviewing, holler if you want me to try to squeeze more.  I tried to do the setenv in one less spot, but it required it being called more frequently in real time.